orttraining/orttraining/test/training_ops/cpu/nn/dropout_op_test.cc

```diff
@@ -146,7 +146,7 @@ TEST(TrainableDropoutTest, EmptyRatio) {
 }
 
 namespace {
-void RunDropoutGradTest(const char* op, float ratio, const std::vector<int64_t>& input_dims, bool default_ratio = true) {
+void RunDropoutGradTest(const char* op, float ratio, const std::vector<int64_t>& input_dims, bool training_mode, bool default_ratio = true) {
```

Comment:
nit: this may be error prone inserting new parameters before the old one. If we forget to update some calls of `RunDropoutGradTest`, previous `default_ratio` will be used as `training_mode`.